cmd/esc/cli/client/client.go

```diff
@@ -133,6 +137,7 @@ type Client interface {
 		ctx context.Context,
 		orgName string,
 		yaml []byte,
+		opts ...CheckYAMLOption,
```

Comment:
Any particular reason to make this a variadic arg rather than a single option struct?

Reply:
*sigh*

Yes. So because of reasons, we can't add required parameters to any methods that are called by the Pulumi CLI. Adding required parameters breaks the ESC CLI build due to a cyclic dependency on the Pulumi CLI.

Ideally this would just be a `bool`.

Reply:
Ouch. Probably worth adding a comment to that effect.

Reply:
Done. Added a comment to the interface as well.